tests/approaches/test_grammar_search_invention_approach.py

```diff
@@ -713,6 +713,7 @@ def test_task_planning_score_function():
         "env": "cover",
         "offline_data_method": "demo+replay",
         "seed": 0,
+        "num_train_tasks": 15,
```

Comment:
lol is this gonna be a problem in a lot of places if we switch to 50

Reply:
possibly, we'll see 😬